src/dotnet-publish-iis/WebConfigTransform.cs

```diff
@@ -57,8 +57,8 @@ private static void TransformHandlers(XElement handlersElement)
 
         private static void TransformHttpPlatform(XElement httpPlatformElement, string appName, bool configureForAzure)
         {
-            var appPath = Path.Combine(configureForAzure ? @"%home%\site" : "..", appName);
-            var logPath = Path.Combine(configureForAzure ? @"\\?\%home%\LogFiles" : @"..\logs", "stdout.log");
+            var appPath = Path.Combine(configureForAzure ? @"%home%\site" : "..", appName).Replace("/", "\\");
+            var logPath = Path.Combine(configureForAzure ? @"\\?\%home%\LogFiles" : @"..\logs", "stdout.log").Replace("/", "\\");
```

Comment:
Should this transform here and on second line above still have the `.log` on them?